### DISCUSSION TITLE
channel element 0.0 is not allocated

### DISCUSSION THREAD

**Discussion by enanto** [SYSTRAN/faster-whisper#411] [Q&A]
Hi all, 
I'm having issues with the transcription of a large .mp4 file. 
I'm using the large-v2 model and the vad_filter, and the code works for any .mp3 file I tested it with. I'm working on Windows 10.

```
model_size = "large-v2"
model = WhisperModel(model_size, device="cpu", compute_type="float32")
segments, _ = model.transcribe(audio_file, language='it', vad_filter=True)
segments = list(segments)
```

When I try to transcribe the mp4 file I get this error: "channel element 0.0 is not allocated" but I can't figure out what's going on. The script keeps running and produces no output, although it reports "Process finished with exit code 0", which seems weird.
The mp4 file is the audio from an 8-hour streamed meeting, around 250 MB. Standard Whisper transcribes it correctly, except for the many hallucinations.
Any help is very much appreciated, thank you.

**Comment by phineas-pta** [comment]
the file maybe corrupted, u can try extract audio then run faster whisper

**Reply by enanto** [reply]
thanks but that's not it. this is already the audio extracted from a video but I have no power over  the format of the file.
I don't think it is corrupted because regular whisper can transcribe it properly.

**Reply by phineas-pta** [reply]
original whisper and faster whisper process audio differently

but why cant u change the format ? like extract to a mp3 or wav

**Reply by enanto** [reply]
oh I see. well i do not have access to the original videos, and the company which owns them will only share the extracted audio in mp4. Do you think converting it to mp3 would help?
